### ISSUE TITLE
Contextualize CSS

### ISSUE THREAD

**Issue by ghost** [BioPhoton/angular-star-rating#74]
## I'm submitting a...
<pre><code>[ ] Bug report
[x] Feature request
</code></pre>

## Current behavior
This module is wrapping style to global scope, exactly as `css-star-rating` does.

Globally appended classes: `.rating`, `.star`, `.star-container`, `.center-all`, `.label-value`.

## Expected behavior
However, angular introduced a very good practice that scope style to a component.

It would be great (to import) to attach SCSS styles to the component but not as global.

A very quick fix is to surround by the angular component name:
```scss
star-rating-comp {
  @import "~css-star-rating/dist/scss/star-rating.scss";
}
```

Probably the better way should be attaching the `css-star-rating`'s style sheet to the `ngComponent`.

## Minimal reproduction of the problem with instructions

For instance, use `<div class="label-value">...</div>` anywhere in your application.

## Environment

<pre><code>
Library version: 3.0.8
Angular version: 5.2.7

Browser:

- [x] all
</code></pre>

**Comment by VivekIshir** [comment]
HI i m seeing only one file inside the assets\images, document saying to import the "star-rating.scss"
Its looking star-rating.scss is missing ? Anybody please help, 
i m using  npm install angular-star-rating --save
![image](https://user-images.githubusercontent.com/34680272/39823586-fc21c4ba-53ca-11e8-96b7-ac8cf096f8db.png)

**Comment by felipefialho** [comment]
I agree with @avBerenger, leak of styles is a big problem of CSS and we can to benefit of Angular scope.

I can help if you want to move on.

**Comment by BioPhoton** [comment]
Hi @LFeh,

I would really appreciate your help!!

Regarding prefixes for css classes I already opened an issues: 
https://github.com/BioPhoton/css-star-rating/issues/33

Let's continue there.

Best, 
Michael
